.github/workflows/upstream_tests.yml

```diff
@@ -379,7 +379,7 @@ jobs:
     env:
       OS: macOS
       PYTHON: ${{ matrix.python-version }}
-      TEST_SPLITS: ${{ matrix.test-splits }}
+      TEST_SPLITS: 3
```

Comment:
`${{ matrix.test-splits}}` is necessary to have 2 unit tests jobs, and 3 integration tests jobs.

Reply:
Oops, this must have leaked in the multiple rebases.